docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,3382 @@
+                               to identify the RP session at the OP when the logout_uri is used. If omitted, the default value is false.
+                backchannel_logout_uri:
+                  type: string
+                  description: RP URL that will cause the RP to log itself out when sent a Logout Token by the OP.
+                backchannel_logout_session_required:
+                  type: boolean
+                  description: Boolean value specifying whether the RP requires that a session ID Claim be included in the Logout Token 
+                          to identify the RP session with the OP when the backchannel_logout_uri is used. If omitted, the default value is false.  
+                request_uris:
+                  type: array
+                  description: Provide a list of request_uri values that are pre-registered by the Client for use at the Authorization Server.
+                  items:
+                    type: string 
+                scopes: 
+                  type: string
+                  deprecated: true
+                  description: This param will be removed in a future version because the correct is 'scope' not 'scopes', see (rfc7591).
+                claims:
+                  type: string
+                  description: String containing a space-separated list of claims that can be requested individually.                  
+                id_token_token_binding_cnf:
+                  type: string
+                  description: Specifies the JWT Confirmation Method member name (e.g. tbh) that the Relying Party expects
+                        when receiving Token Bound ID Tokens. The presence of this parameter indicates that the Relying Party 
+                        supports Token Binding of ID Tokens.
+                        If omitted, the default is that the Relying Party does not support Token Binding of ID Tokens.                  
+                tls_client_auth_subject_dn:
+                  type: string
+                  description: An string representation of the expected subject distinguished name of the certificate, 
+                             which the OAuth client will use in mutual TLS authentication.                  
+                allow_spontaneous_scopes:
+                  type: boolean
+                  description: Specifies whether to allow spontaneous scopes for client. The default value is false.
+                  example: false      
+                spontaneous_scopes:
+                  type: array
+                  description: List of spontaneous scopes 
+                  items:
+                    type: string
+                run_introspection_script_before_access_token_as_jwt_creation_and_include_claims:
+                  type: boolean
+                  description: Boolean value with default value false. If true and access_token_as_jwt=true then run introspection script 
+                              and 
```

Comment:
describe permission object https://docs.kantarainitiative.org/uma/wg/rec-oauth-uma-federated-authz-2.0.html#uma-bearer-token-profile

Reply:
The screenshot seems mismtach...
I have added description in /uma2-configuration endpoint for of permission_endpoint, resource_registration_endpoint and introspection_endpoint
 description: The endpoint URI at which the resource server requests permissions on the client's behalf.

Reply:
I gave you link to how permission object look in json. You have to define it in swagger spec, not just `type: object`. It doesn't say anything about internal structure of the object. It NOT about `description`, it's about missed definition of the permission object.

Reply:
Done
![image](https://user-images.githubusercontent.com/43700552/76422997-26950500-63cc-11ea-95fe-ddb8002d57a8.png)